spec/requests/api/restaurants_endpoint_spec.rb

```diff
@@ -50,18 +50,37 @@
   end
 
   describe '#show' do
+    context 'with anonymous doubles' do
+      let!(:menus) do
+        3.times {create(:menu)}
+      end
 
-    context 'with a specific restaurant' do
+      it 'returns a collection of menus' do
+        get '/api/restaurants'
+        expect(JSON.parse(response.body)['data'].count).to eq 3
+      end
+    end
+
+    context 'a menu within a specific restaurant' do
       let(:category) {create(:restaurant_category, name: 'Thai')}
-      let!(:thai_food) {create(:restaurant,
-                               name: 'Thai Palace',
-                               description: 'Lovely place.',
-                               city: 'Gothenburg',
-                               street_address: 'Holtermansgatan 1C',
-                               post_code: '410 29',
-                               restaurant_category: category)}
-      let!(:menu_lunch) {create(:menu, name: 'lunch', restaurant: thai_food)}
-      let!(:product_category) {create(:product_category, name: 'Main', menu: menu_lunch, restaurant: thai_food)}
+      let!(:french_food) {create(:restaurant,
```

Comment:
Including the **restaurant category + restaurant** as these are required for creating a **menu, product category and product**. Have tried to create only the menu with an associated product category resulting in a nil obj.